docs/en/writing-running-appium/other/troubleshooting.md

```diff
@@ -141,6 +141,14 @@ own items in the SDK manager. Make sure you install the build-tools and platform
     - If it returns `69`, you should call `xcodebuild -runFirstLaunch` once to prevent the root error, `Install Started
 Install Failed: Error Domain=DVTDownloadableErrors Code=3 "The authorization was denied since no user interaction was possible." UserInfo={AuthorizationErrorCode=-60007, NSLocalizedDescription=The authorization was denied since no user interaction was possible.}`
     - example: https://github.com/appium/WebDriverAgent/pull/165#discussion_r281458167
+* Make sure your `.ipa` had no Unicode file names
+    - Appium unzip `.ipa` to install packed the application under test, but [the macOS's Archive Utils has an issue](https://stackoverflow.com/questions/13261347/correctly-decoding-zip-entry-file-names-cp437-utf-8-or). So, unzip generates could generate wrong file names.
```

Comment:
Appium unzips .ipa packages to be able to install the app under test, but there a known issue with Mac OS ZIP library